lib/commands/file-movement.js

```diff
@@ -210,7 +222,7 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
     await mkdirpDevice(service, path.dirname(relativePath));
     const stream = await service.createWriteStream(relativePath, { autoClose: true });
     stream.write(Buffer.from(base64Data, 'base64'));
-    const closeEvent = new B((resolve) => stream.on('close', resolve));
+    const closeEvent = new B((resolve) => stream.on('close', resolve)).timeout(IO_TIMEOUT);
```

Comment:
should we cover this into try/catch block to show appropriate timeout error?

same below

Reply:
Makes sense